src/app/cube/new-home/learning-object-info/hierarchies/hierarchies.component.ts

```diff
@@ -1,15 +1,26 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input} from '@angular/core';
+import { ClientRequest } from 'http';
 
 @Component({
   selector: 'clark-hierarchies',
   templateUrl: './hierarchies.component.html',
   styleUrls: ['./hierarchies.component.scss']
 })
 export class HierarchiesComponent implements OnInit {
+  @Input() length: 'nanomodule' | 'micromodule' | 'module' | 'unit' | 'course';
```

Comment:
```suggestion
  length: 'nanomodule' | 'micromodule' | 'module' | 'unit' | 'course';
```
This variable isn't being dynamically set so we don't have to use the @Input decorator